apps/apps.humanatlas.io/src/app/pages/us6/web-components.component.ts

```diff
@@ -0,0 +1,49 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ProductLogoComponent } from '@hra-ui/design-system/product-logo';
+import { SoftwareStatusIndicatorComponent } from '@hra-ui/design-system/software-status-indicator';
+import { MatFormField, MatLabel } from '@angular/material/form-field';
+import { MatOption } from '@angular/material/core';
+import { MatSelect } from '@angular/material/select';
+import organs from '../../../../../apps.humanatlas.io/organs.json';
+import components from '../../../../../apps.humanatlas.io/component-defs.json';
+import { WebComponentCardComponent } from '@hra-ui/design-system/web-component-card';
+import { ButtonsModule } from '@hra-ui/design-system/buttons';
+@Component({
+  selector: 'hra-web-components',
+  standalone: true,
+  imports: [
+    CommonModule,
+    WebComponentCardComponent,
+    ProductLogoComponent,
+    SoftwareStatusIndicatorComponent,
+    ButtonsModule,
+    MatFormField,
+    MatOption,
+    MatLabel,
+    MatSelect,
+  ],
+  templateUrl: './web-components.component.html',
+  styleUrl: './web-components.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class WebComponentsComponent {
+  public organs = organs;
```

Comment:
No need to mark as `public` as it it the default. Please add `protected readonly` instead